src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/VisitSchedulerExceptionHandler.kt

```diff
@@ -87,6 +88,18 @@ class VisitSchedulerExceptionHandler(
     return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(error)
   }
 
+  @ExceptionHandler(VisitToMigrateException::class)
```

Comment:
Not very clear on the migration process - this exception when thrown will impact only 1 visit and the migration process will continue. Is that correct?

Reply:
yes thats correct,... each visit is migrated one at a time,.. and associated info is given for syscon